go/teams/chain.go

```diff
@@ -1809,3 +1849,25 @@ func (t *TeamSigChainPlayer) assertIsSubteamID(subteamIDStr string) (keybase1.Te
 	}
 	return subteamID, nil
 }
+
+func (t *TeamSigChainPlayer) parseTeamSettings(settings *SCTeamSettings, newState *TeamSigChainState) error {
```

Comment:
This allows `options.JoinAs` to remain unspecified, in which case it will default to `Role_NONE`. Which seems nonsensical. So maybe that shouldn't be allowed.

Reply:
You're right, it's also not allowed on the server. The idea at first was to be able to do things like:
Link 1: Set team to open + join_as: reader
Link 2: Set team to closed
Link 3: Set team to open (without specifying join_as)

And after 3 it would be open and join_as: reader ("inherited" from 1st link).

Reply:
This should be fixed now.